frontend/src/Pages/ReleasePlan.jsx

```diff
@@ -85,162 +95,251 @@ const ReleasePlan = ({ projectId }) => {
     } catch { }
   }
 
-	function createNewSprints() {
-		console.log("creating new");
-		var options = {
-			method: "POST",
-			credentials: "include",
-			headers: {
-				"Content-Type": "application/json",
-			},
-			body: JSON.stringify({ sprintNumber: sprints.length + 1 }),
-		};
+  //updates the problem statement
+  function editProblem(e) {
+    var options = {
+      method: "POST",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ problemStatement: e.target.value },
+        { goalStatement: e.target.value }),
+    };
+    fetch(`http://localhost:8080/api/release/${releaseId}/edit`, options)
+      .then((result) => {
+        if (result.status === 200) {
+          result.json().then((response) => {
+            setProblem(response.problemStatement);
+          });
+        }
+      })
+  }
+
+  //updates the HighLevelGoals
```

Comment:
Top comments should have a space after //

But also these comments are redundant. Its clear what these functions do